templates/.kokoro/presubmit/graalvm-sub-jobs/native-17/common.cfg

```diff
@@ -0,0 +1,68 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build configs are duplicated in native/ and native-17/
+# Build configs don't inherit the hierarchical common.cfg: qaqs/6158654898243108864
```

Comment:
FYI, job.proto inherits hierarchical common.cfg but build.proto doesn't. Another option is to have one common.cfg in presubmit/ and have all 40 .cfg files in presubmit/ so we don't have to maintain two common.cfg